src/main/java/Iris.java

```diff
@@ -0,0 +1,56 @@
+import task.Task;
+import static method.Command.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Iris {
+    static ArrayList<Task> tasks;
+
+    public static void printDivider() {
+        System.out.println("---------------------------------------------");
+    }
+
+    public static boolean chat(String text) {
+        String command = text.split(" ")[0].toLowerCase();
+        printDivider();
+        switch (command) {
+        case "":
+            System.out.println("Tell me your needs! I'm here to help!");
+            break;
+        case "bye":
```

Comment:
Would it be possible to remove the need of magic strings and numbers in your entire code base?